modules/nextflow/src/main/groovy/nextflow/k8s/client/K8sClient.groovy

```diff
@@ -500,6 +503,12 @@ class K8sClient {
             def msg = "K8s pod '$podName' execution failed"
             if( status.reason ) msg += " - reason: ${status.reason}"
             if( status.message ) msg += " - message: ${status.message}"
+            if ( 'OutOfcpu' == status.reason && (status.message as String).startsWith("Pod Node didn't have enough resource: cpu,")) {
```

Comment:
Is the message check needed if the reason already matches?

Reply:
I wanted to be 100% sure, but the reason seemed unique, so I adjusted it.